dipy/core/gradients.py

```diff
@@ -210,7 +211,26 @@ def bvecs(self):
         denom = self.bvals + (self.bvals == 0)
         denom = denom.reshape((-1, 1))
         return self.gradients / denom
-
+    
+    def getitem(self, bmin=0, bmax=np.inf):
```

Comment:
here it should be __get_item__(self, idx):

Reply:
Sure, I'll change this